mutation_partition.cc

```diff
@@ -304,17 +304,19 @@ mutation_partition::apply(const schema& s, const mutation_fragment& mf) {
     mf.visit(applier);
 }
 
-stop_iteration mutation_partition::apply_monotonically(const schema& s, mutation_partition&& p, cache_tracker* tracker, is_preemptible preemptible) {
+std::tuple<stop_iteration, mutation_application_stats> mutation_partition::apply_monotonically(const schema& s, mutation_partition&& p, cache_tracker* tracker, is_preemptible preemptible) {
 #ifdef SEASTAR_DEBUG
```

Comment:
Again would be better as an output parameter. The stop_iteration is the main point of the function, the stats are a side effect.

Reply:
Done.